versions.cfg

```diff
@@ -43,4 +43,11 @@ repoze.who = 2.2
 RestrictedPython = 3.6.0
 zope.sequencesort = 4.0.1
 zope.testbrowser = 3.11.1
+ZODB = 4.2.0
+sphinx-rtd-theme = 0.1.9
+alabaster = 0.7.7
+Babel = 2.2.0
+snowballstemmer = 1.2.1
```

Comment:
yes, they are. Probably it makes sense to keep these dependencies in an entry point of the egg and mark them in the versions.cfg